src/config.ts

```diff
@@ -12,28 +12,32 @@ export const getOpenAIApiKey = (): string => {
   if (!process.env.OPENAI_API_KEY) {
     throw new Error("OPENAI_API_KEY is not set");
   }
+
   return process.env.OPENAI_API_KEY;
 };
 
 export const githubToken = (): string => {
   if (!process.env.GITHUB_TOKEN) {
     throw new Error("GITHUB_TOKEN is not set");
   }
+
   return process.env.GITHUB_TOKEN;
 };
 
 export const getGitHubEnvVariables = (): Record<string, string> => {
-  const missingVars = ["GITHUB_SHA", "BASE_SHA", "GITHUB_TOKEN"].filter(
-    (varName) => !process.env[varName]
-  );
+  const envVars = ["GITHUB_SHA", "BASE_SHA", "GITHUB_TOKEN"];
+  const missingVars: string[] = [];
+  envVars.forEach((envVar) => process.env[envVar] ?? missingVars.push(envVar));
+
   if (missingVars.length > 0) {
     logger.error(`Missing environment variables: ${missingVars.join(", ")}`);
     throw new Error("One or more GitHub environment variables are not set");
   }
+
   return {
-    githubSha: process.env.GITHUB_SHA as string,
-    baseSha: process.env.BASE_SHA as string,
-    githubToken: process.env.GITHUB_TOKEN as string,
+    githubSha: process.env.GITHUB_SHA ?? '',
+    baseSha: process.env.BASE_SHA ?? '',
+    githubToken: process.env.GITHUB_TOKEN ?? '',
```

Comment:
[Note] This isn't a perfect fix, but we shouldn't get here if any of the env variables are undefined and it felt like I was taking too long on this 😄

Reply:
Not a huge fan of this. Works for now I guess.

Reply:
Neither am I :' ) I will fix in my casting ticket